EventHandlers/PrebidMobileMoPubAdapters/Sources/MoPubMediationUtils.swift

```diff
@@ -89,18 +93,17 @@ public class MoPubUtils {
         
         adObject.setValue(extras, forKey: MoPubSelector_localExtras)
     }
-
+    
     /**
      Puts to ad object's localExtra the ad object (winning bid or native ad) and configId
      and populates adObject's keywords by targeting info
      @return YES on success and NO otherwise (when the passed ad has wrong type)
      */
-    public static func setUpAdObject(_ adObject: NSObject,
-                                     configID:String,
-                                     targetingInfo: [String : String],
-                                     extraObject:Any?,
-                                     forKey:String ) -> Bool {
```

Comment:
Let's return this param back. Right now we use only one key since we do not support Native. But are going to support native ASAP. In addition, it might be useful for custom mediation platforms eventually.